smriprep/workflows/base.py

```diff
@@ -129,6 +133,8 @@ def init_smriprep_wf(
                 spaces=[s for s in output_spaces.keys() if s.startswith('fsaverage')] + [
                     'fsnative'] * ('fsnative' in output_spaces)),
             name='fsdir_run_%s' % run_uuid.replace('-', '_'), run_without_submitting=True)
+        if fs_subjects_dir is not None:
+            fsdir.inputs.subjects_dir = str(fs_subjects_dir.absolute())
```

Comment:
Is `absolute()` better than `resolve()` for some reason? (this is a question, not a change suggestion here).

Reply:
It doesn't attempt to resolve symlinks, just turns relative paths into absolute.

Reply:
Thanks for the clarification - pretty useful.